package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pins-data-model",
-  "version": "1.7.2",
+  "version": "1.7.3",
```

Comment:
@fabfab-kc 
I think I did it already this morning, did you see an old copy, or is there more to do?

Reply:
sorry must have missed it!